api/cpp/esp-idf/slint/src/slint-esp.cpp

```diff
@@ -347,29 +347,11 @@ void slint_esp_init(slint::PhysicalSize size, esp_lcd_panel_handle_t panel,
         .touch_handle = touch ? *touch : nullptr,
         .buffer1 = buffer1,
         .buffer2 = buffer2,
-        // For compatibility with earlier versions of Slint, we compute the value of
-        // byte_swap the way it was implemented in Slint (slint-esp) <= 1.6.0:
         .byte_swap = buffer2.has_value()
```

Comment:
```suggestion
        // For compatibility with earlier versions of Slint, we compute the value of
        // byte_swap the way it was implemented in Slint (slint-esp) <= 1.6.0:
        .byte_swap = !buffer2.has_value()
```

Reply:
But the comment is wrong, this is not what  1.6 was doing, that's why I removed it.